frontend/src/components/messstelle/charts/MessstelleDiagramme.vue

```diff
@@ -17,22 +17,24 @@
                 Belastungsplan
                 <v-icon>mdi-arrow-decision</v-icon>
             </v-tab>
-            <v-tab>
-                Ganglinie
-                <v-icon>mdi-chart-histogram</v-icon>
-            </v-tab>
-            <v-tab>
-                Listenausgabe
-                <v-icon>mdi-table</v-icon>
-            </v-tab>
-            <v-tab>
-                Heatmap
-                <v-icon>mdi-chart-bubble</v-icon>
-            </v-tab>
-            <v-tab>
-                Zeitreihe
-                <v-icon>mdi-timer-sand</v-icon>
-            </v-tab>
+            <tooltipTab
+                :is-disabled="isBiggerThanFiveYears"
+                :tooltip-text="ZEITRAUM_GROESSER_FUENF_JAHRE"
+                icon="mdi-chart-histogram"
+                tab-name="Ganglinie"
+            />
```

Comment:
Wieso so kompliziert? Einfach im Tab-Item anstelle der Card einen Banner einblenden. Dann sieht man die Nachricht sofort. Siehe https://github.com/it-at-m/dave-admin-portal/blob/sprint/frontend/src/components/messstelle/UpdateMessstelleForm.vue

Reply:
ehrlich gesagt find ich einfach das es so besser ausschaut und direkt klar wird das diese anzeigen nicht zur verfügung steht. Wenn du willst kann ich es auch auf ein banner abändern

Reply:
Ich persönlich fände ein Banner schöner. Aber letztendlich hat der Fachbereich das letzte Wort. Daher frag da bitte mal nach.